ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -11343,27 +11343,29 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         {
                             b1.Property<int>("NotificationId");
 
-                            b1.Property<int>("AdultsActiveTB");
+                            b1.Property<int?>("AdultsActiveTB");
 
-                            b1.Property<int>("AdultsFinishedTreatment");
+                            b1.Property<int?>("AdultsFinishedTreatment");
 
-                            b1.Property<int>("AdultsLatentTB");
+                            b1.Property<int?>("AdultsIdentified");

```

Comment:
old context seemed to not have AdultsIdentified on it somehow but this change should be correct (it's also causing a merge conflict here)

Reply:
Yeah, that looks fine - the merge conflict may have been from a different branch correcting it?
Let me know if you want another pair of eyes when resolving the merge conflict, but I think it looks right :)

Reply:
This line was also missing for me, but it all works now :)